src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesDeployerProperties.java

```diff
@@ -464,6 +464,59 @@ public void setEnvironmentVariables(String[] environmentVariables) {
 		}
 	}
 
+	/**
+	 * The {@link RestartPolicy} to use. Defaults to {@link RestartPolicy#Never}.
+	 */
+	private RestartPolicy restartPolicy = RestartPolicy.Never;
+
+	/**
+	 * The default service account name to use for tasks.
+	 */
+	protected static final String DEFAULT_TASK_SERVICE_ACCOUNT_NAME = "default";
+
+	/**
+	 * Service account name to use for tasks, defaults to:
+	 * {@link #DEFAULT_TASK_SERVICE_ACCOUNT_NAME}
+	 */
+	private String taskServiceAccountName = DEFAULT_TASK_SERVICE_ACCOUNT_NAME;
```

Comment:
I believe the changes from this PR can fix this as well: https://github.com/spring-cloud/spring-cloud-dataflow/issues/4619